specification/network/resource-manager/Microsoft.Network/stable/2024-10-01/virtualNetworkGateway.json

```diff
@@ -4192,6 +4242,31 @@
       },
       "description": "List of virtual network gateway routes."
     },
+    "RadiusAuthServer": {
+      "properties": {
+        "radiusServerAddress": {
```

Comment:
>value [](http://example.com/codeflow?start=9&length=5)

Declare the nextLink property also - even if your RP doesn't return it, ARM reserves the right to rewrite your response and add one.

Reply:
Added nextLink property to RadiusAuthServerListResult